JavaSample/src/main/java/processor/Message.java

```diff
@@ -0,0 +1,13 @@
+package processor;
+
+public class Message {
+    private final String text;
+
+    public Message(String text) {
```

Comment:
Just a suggestion but maybe use getter and constructor from lombok? https://projectlombok.org/features/GetterSetter

Reply:
I use Lombok as the first time. However, I encounter the issue to configure the lombok of gradle ver 5.x. However, I solved this now.